tests/unit/GoFields/GoFieldInput.spec.js

```diff
@@ -1,25 +1,47 @@
 import { createLocalVue, shallowMount } from '@vue/test-utils'
 import * as AgoUikit from '../../../src'
 import { GoFieldInput } from '../../../src'
-import renderCases from './GoFieldInput.render-cases'
 
 const localVue = createLocalVue()
-
 localVue.use(AgoUikit)
 
 describe('GoFieldInput', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = shallowMount(GoFieldInput, {
+      localVue,
+      propsData: {
+        value: 'first name'
+      }
+    })
+  })
+
   // Install
   it('should be installed', () => {
     expect(localVue.options.components['GoFieldInput']).toBeTruthy()
   })
 
-  // Render testing
-  // renderCases.forEach(renderCase => {
-  //   const [fromComp, toHtml] = renderCase
+  it('should have errors bag present', () => {
+    expect(wrapper.vm.errors.items).toBeDefined()
+  })
+
+  it('should have not vee-validate errors register', () => {
+    expect(wrapper.vm.errors.items).toEqual([])
+  })
+
+  it('should have input value equal to v-model pass in props', () => {
```

Comment:
`should have input value equal to v-model passed in (the name of the prop) prop`